framework/src/controls/WebServerControlTypeRegistry.C

```diff
@@ -0,0 +1,19 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "WebServerControlTypeRegistry.h"
+
+WebServerControlTypeRegistry &
+WebServerControlTypeRegistry::getRegistry()
+{
+  static WebServerControlTypeRegistry * registry_singleton = nullptr;
+  if (!registry_singleton)
+    registry_singleton = new WebServerControlTypeRegistry();
```

Comment:
```suggestion
    registry_singleton = new WebServerControlTypeRegistry;
```
default-initialize (does not functionally make a difference in this case because `WebServerControlTypeRegistry` has a default CTOR, but it made me do a double take)